pkg/controller/build/utils/helpers.go

```
@@ -77,6 +77,10 @@ func GetEtcRegistriesConfigMapName(mosb *mcfgv1.MachineOSBuild) string {
 	return fmt.Sprintf("etc-registries-%s", getFieldFromMachineOSBuild(mosb))
 }
 
+func GetEtcRegistriesDConfigMapName(mosb *mcfgv1.MachineOSBuild) string {
```

golangci-lint (2.12.2)

[warning] exported: exported function GetEtcRegistriesDConfigMapName should have comment or be unexported

(revive)